packages/n8n-nodes-honeybook/package.json

```diff
@@ -0,0 +1,127 @@
+{
+  "name": "n8n-nodes-honeybook",
+  "version": "1.26.0",
+  "description": "honeyBook n8n nodes",
+  "main": "index.js",
+  "scripts": {
+    "clean": "rimraf dist .turbo",
+    "dev": "pnpm watch",
+    "typecheck": "tsc",
+    "build": "tsc -p tsconfig.build.json && tsc-alias -p tsconfig.build.json && gulp build:icons && gulp build:translations && pnpm build:metadata",
+    "build:translations": "gulp build:translations",
+    "build:metadata": "pnpm n8n-generate-known && pnpm n8n-generate-ui-types",
+    "format": "prettier --write . --ignore-path ../../.prettierignore",
+    "lint": "eslint . --quiet && node ./scripts/validate-load-options-methods.js",
+    "lintfix": "eslint . --fix",
+    "watch": "tsc-watch -p tsconfig.build.json --onCompilationComplete \"tsc-alias -p tsconfig.build.json\" --onSuccess \"pnpm n8n-generate-ui-types\"",
+    "test": "jest"
+  },
+  "files": [
+    "dist"
+  ],
+  "n8n": {
```

Comment:
only need to update this to export new nodes and credentials - only things here will actually be exported - note that if something is here and is not defined in code, build will fail.